package.json

```diff
@@ -10,6 +10,9 @@
     "prepare": "husky"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.4",
```

Comment:
We are using TailwindCSS utility classes for styling. It's best not to introduce more CSS implementation in the project.